sirepo/package_data/template/ml/scale.py.jinja

```diff
@@ -9,11 +10,10 @@ from sklearn.preprocessing import {{ dataFile_outputsScaler }}
 {% endif %}
 
 def read_data(path, **kwargs):
-    return np.genfromtxt(
+    return sirepo.numpy.ndarray_from_csv(
         path,
-        delimiter=',',
-        skip_header={{ 1 if columnInfo_hasHeaderRow else 0 }},
-        **kwargs
+        {{ 1 if columnInfo_hasHeaderRow else 0 }},
```

Comment:
`int(columnInfo_hasHeaderRow)` is proper Python. `bool(True) == 1`.

Reply:
bool is just fine de563001.

What's up with folks using 1 and 0 in context where a truthy or falsey value is what is need. I know py.path and numpy do this in their docs. Just shorter? Do you have a preference?

Reply:
Well looking at numpy I was wrong. An int is required. `int()` is undefined in jinja. Reverted f5798c81

Question still holds for py.path (ex the rec kwarg in chmod())

Reply:
Playing around I guess an int isn't technically required in this context, at least in what I tried. Python implicitly casts it (ex `1 + True == 2` and `for x in range(True): print(x) # Prints 0`.

Reply:
Whoops int(True) == 1. :)

Whoops on not noticing jinja which is not python.

Reply:
> What's up with folks using 1 and 0 in context where a truthy or falsey value is what is need. I know py.path and numpy do this in their docs. Just shorter? Do you have a preference?

It's messed up is what it is. :)  I think if a language has True, use it. Many people define TRUE in C, including [numpy](https://github.com/numpy/numpy/blob/master/numpy/core/include/numpy/npy_common.h#L358) (actually  NPY_TRUE and [TRUE](https://github.com/numpy/numpy/blob/master/numpy/core/include/numpy/noprefix.h#L147)).

There are reasons to use True, instead of 1, e.g. it maps properly in JSON.

Reply:
Ok:) JSON seems like a pretty good reason to use bool, especially in our context. Thanks.